lom/int_test/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
Suggest push this integration test via a different PR.
Because the way we run the test is completely different from your current way.
See comments below.
Till then, you may keep this in a private branch
By EOW, LoM is running in container. Your test files are available as tar archive inside container.
All we do:
docker exec -it device-health "tar -xzvf ... /tmp/"
docker run -it device-health "/tmp/test/run.sh"

Here the needed engine & pluginMgr are available in /usr/bin/

Reply:
As agreed , will revisit integration tests later